nullable/conversions.go

```diff
@@ -0,0 +1,45 @@
+package nullable
+
+import (
+	"time"
+)
+
+// ToString returns a new String
+func ToString(s *string) String {
```

Comment:
Is `NewString(s *string)` more idiomatic for constructors? 

Same for all funcs in this file.

Reply:
I agree with this one. That does seem more idiomatic.

Reply:
New is used when you return a pointer to something, in this case we are not, so I did not feel it was appropriate

Reply:
`Make`

Reply:
Yes it should be, I'll update

Reply:
Here is a couple examples that seem to support 'Make'
Returning an interface: https://github.com/golang/go/blob/44286d09c5fcdb703624fad6ce24306294cab1fe/src/go/constant/value.go#L269
Returning a value: https://github.com/golang/go/blob/44286d09c5fcdb703624fad6ce24306294cab1fe/src/cmd/internal/src/pos.go#L37